CONTRIBUTION.md

```diff
@@ -0,0 +1,223 @@
+fixed, may be suspect.  Ditto for new features that can't prove they actually
+work.
+
+Your ideally include unit tests and/or storybooks to test and help describe what you are solving
+
+We've found that test-first development really helps make features better
+architected and identifies potential edge cases earlier instead of later.
+Writing tests before the implementation is strongly encouraged.
+
+Full example
+------------
+
+Here's an example workflow for a project ``theproject`` hosted on Github, which
+is currently in version 1.3.x. Your username is ``yourname`` and you're
+submitting a basic bugfix.
+
+Preparing your Fork
+^^^^^^^^^^^^^^^^^^^
+
+1. Click 'Fork' on Github, creating e.g. ``yourname/theproject``.
+2. Clone your project: ``git clone git@github.com:yourname/theproject``.
+3. ``cd theproject``
+4. Install the development requirements: ``npm install``.
+5. Create a branch: ``git checkout -b foo-the-bars 1.3``.
+
+Making your Changes
+^^^^^^^^^^^^^^^^^^^
+
+1. Add changelog entry crediting yourself.
+2. Write tests expecting the correct/fixed functionality; make sure they fail.
+3. Hack, hack, hack.
+4. Run tests again, making sure they pass.
+5. Commit your changes: ``git commit -m "Foo the bars"``
+
+Creating Pull Requests
+^^^^^^^^^^^^^^^^^^^^^^
+
+1. Push your commit to get it back up to your fork: ``git push origin HEAD``
+2. Visit Github, click handy "Pull request" button that it will make upon
+   noticing your new branch.
+3. In the description field, write down issue number (if submitting code fixing
+   an existing issue) or describe the issue + your fix (if submitting a wholly
+   new bugfix).
+4. Hit 'submit'! And please be patient - the maintainers will get to you when
+   they can.
+
+## Setup guide
+
+### Cloning repository
+
+Start by cloning the repository to desired directory.
+
+```
+git clone https://github.com/equinor/esv-intersection.git
+```
+
+### Install dependencies
+
+Install all dev dependencies defined in package.json using node.
+
+```
+npm install
+```
+
+## Usage
+
+### Storybook
+
+```
+npm run storybook
+```
+
+### Document generation
+
+```
+npm run docs
+```
+
+This command will overwrite any old documentation
+
+### Creating tests
+
+All tests are defined within the test-folder. Jest naming convention is to place tests for SOMESCRIPT.ts in a single file SOMESCRIPT.test.ts.
+
+### Testing
+
+```
+npm run test
+```
+
+Executes all tests defined within the test folder.
+
+```
+npm run test:watch
+```
+
+Executes all tests, but does not return immediately. Makes it possible to re-run failed tests quickly.
+
+### Building/Compiling
+
+```
+npm run build
+```
+
+Compiles the code found within the src-folder. Build is outputted to a new dist-folder.
+
+----
+The content of this file is largely inspired by [bitprophet's contribution guide](https://github.com/bitprophet/contribution-guide.org/blob/master/index.rst).
+This file is licensed under the _BSD 2-Clause "Simplified" License_ that follows:
```

Comment:
What if we move these two lines at the beginning of the file, and leave the license here at the end?